packages/core/src/service/services/session.service.ts

```diff
@@ -146,8 +146,11 @@ export class SessionService implements EntitySubscriberInterface {
      * Serializes a {@link Session} instance into a simplified plain object suitable for caching.
      */
     serializeSession(session: AuthenticatedSession | AnonymousSession): CachedSession {
-        const expiry =
-            Math.floor(new Date().getTime() / 1000) + this.configService.authOptions.sessionCacheTTL;
+        const { sessionCacheTTL } = this.configService.authOptions;
+        const sessionCacheTTLSeconds =
+            typeof sessionCacheTTL === 'string' ? ms(sessionCacheTTL) / 1000 : sessionCacheTTL;
```

Comment:
@michaelbromley should we Math.floor the whole expression assigned to `sessionCacheTTLSeconds` just like it's done with `new Date().getTime() / 1000`?

if so we should have some minimum for `sessionCacheTTLSeconds` like 1 to not run into the case of `sessionCacheTTLSeconds` being 0

if not, what is the reason to use `Math.floor` on `new Date().getTime() / 1000` anyway?

Reply:
Honestly I'm not sure if there is any good reason to do the Math.floor at all. I mean, it keeps the `cacheExpiry` value to a round number of seconds, but I don't think that really makes any functional difference. So let's just get rid of it.